set.mm

```diff
@@ -255823,8 +256323,10 @@ is applicable for semirings (without using the commutativity of the
        the additive structure must be abelian (see ~ ringcom ), care must be
        taken that in the case of a non-unital ring, the commutativity of
        addition must be postulated and cannot be proved from the other
-       conditions.  (Contributed by NM, 18-Oct-2012.)  (Revised by Mario
-       Carneiro, 27-Dec-2014.) $)
+       conditions.  Therefore, it can be shown that a unital ring is a
+       non-unital ring ( ~ ringrng ) only after ~ringabl was proven.
```

Comment:
```suggestion
       non-unital ring ( ~ ringrng ) only after ~ ringabl was proven.
```

Reply:
This was fixed in a follow up commit already.